### DISCUSSION TITLE
Can't start an example app with 'npm init next-app '

### DISCUSSION THREAD

**Discussion by gisete** [vercel/next.js#12103] [Help] [answered]
Running node v10. I run `npm init next-app`  and all I get is a starter package.json file created.
Am I missgin something?

**Comment by ankit-gupta1307** [comment]
Same thing is happening for me.

**Comment by Gabr13dev** [comment]
Same here too, I think we forgot something.

**Comment by ankit-gupta1307** [comment]
Nooo, there is nothing to forgot, they metioned  simple comment but it's not working.

**Reply by ankit-gupta1307** [reply]
@Gabr13dev

**Comment by Timer** [comment]
Just confirmed this is working fine. Please elaborate more, as no error message was provided.

**Comment by ankit-gupta1307** [comment]
After running this command it's working like as I have done only npm init ----> npm init next-app nextjs-blog --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter". Please refer to the screenshot @Timer 

[
![Screenshot (60)](https://user-images.githubusercontent.com/17529193/83427203-fcdcfe80-a44d-11ea-9cd7-cda8c4fb55e3.png)
](url)

**Reply by ankit-gupta1307** [reply]
And it's creating only package.json file. Also tried deleting the the text after example as mentioned in the website.

**Comment by gisete** [comment]
Maybe the documentation was in the process of being updated.

I can confirm that using `npx create-next-app`, as it's being recommended now, works 👍

**Comment by Timer** [comment, marked as answer]
Fixed via https://github.com/vercel/next.js/pull/13637 for users on older npm versions.
